core/metacling/src/TCling.cxx

```diff
@@ -1138,6 +1137,10 @@ static void LoadCoreModules(cling::Interpreter &interp)
 
    if (!LoadModule(moduleMap.findModule("RIO")->Name, interp))
       Error("TCling::LoadCodeModule", "Cannot load module RIO");
+
+   assert(interp.getMacro("gROOT") && "Couldn't load gROOT macro?");
+   interp.declare("#include <cassert>\n");
+   interp.declare("#ifdef I\n #undef I\n #endif\n");
```

Comment:
Could we move that in the else part of case `LIBM`? Probably we can use a PP API to undefine `I` and avoid recursive parsing.